docs/docs/limits/README.md

```diff
@@ -89,10 +89,6 @@ Currently, most of the [limits that apply to HTTP triggers](#http-triggers) also
 
 The only limit that differs between email and HTTP triggers is the payload size: the body of HTTP requests is limited to `{{$site.themeConfig.PAYLOAD_SIZE_LIMIT}}`, where the total size of an email sent to a workflow - its body, headers, and attachments - is limited to `{{$site.themeConfig.EMAIL_PAYLOAD_SIZE_LIMIT}}` on the default interface.
 
-**However, you can send emails up to `30MB` in size using the address `[YOUR EMAIL ENDPOINT]@upload.pipedream.net`**. [Read more here](/workflows/steps/triggers/#sending-large-emails).
-
-This `30MB` limit cannot be raised.
```

Comment:
Keep this since we've received questions about it. Some limits (like QPS) can be raised and others cannot, so we like to call that out. We need a better matrix of this [like AWS has](https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-limits.html).